src/components/RadioButton/RadioButton.vue

```diff
@@ -1,37 +1,52 @@
 <template>
   <div
     :class="[
-      'inline-block mr-4 mt-1',
-      { 'cursor-not-allowed': disabled || readonly }
+      'inline-block mr-6 mt-1',
+      {'!block mb-1': parent.props.separateLines}
```

Comment:
I really don't think we should ever be referencing the `parent` from a child component.  This depends too much on knowledge of the implementation of the parent which could easily change and then break things.  So we need to come up with a solution that's more flexible.  We could add a prop to this component that the parent sets when its `separateLines` is `true`.  Though I actually think probably this can all be accomplished with styling on the parent.  I'll take a look and see if I can offer a suggestion.

Reply:
I think this might be a merge conflict gone wrong, I took this out [here](https://github.com/lob/ui-components/pull/264)

Reply:
Oh yes, looks like it, good catch!